src/diffpy/labpdfproc/tests/test_tools.py

```diff
@@ -1,78 +1,74 @@
-import argparse
-import os
 import re
 from pathlib import Path
 
 import pytest
 
+from diffpy.labpdfproc.labpdfprocapp import get_args
 from diffpy.labpdfproc.tools import known_sources, set_output_directory, set_wavelength
 
 params1 = [
-    ([None], ["."]),
-    (["."], ["."]),
-    (["new_dir"], ["new_dir"]),
-    (["existing_dir"], ["existing_dir"]),
+    ([], ["."]),
+    (["-o", "."], ["."]),
```

Comment:
in tests I prefer that we always use the long name `--output_dir` or whatever it is, to make the test more readable.

Reply:
I think the string here needs to be one of the arguments for user to specify? For example user specifies -o instead of output_directory. If we put output_directory it will cause an error saying it's not one of the arguments. Not sure if what I say here completely makes sense.